$NetBSD: patch-ar_acpyacc_y,v 1.1 2012/09/28 08:59:07 dholland Exp $

Fix warnings about old-style function definitions that are fatal in
some(?) builds.

--- ar/acpyacc.y~	2011-11-10 06:56:46.000000000 +0000
+++ ar/acpyacc.y
@@ -414,7 +414,7 @@ arscp_extract(struct list *list)
 
 /* List modules of archive. (Simple Mode) */
 static void
-arscp_list()
+arscp_list(void)
 {
 
 	if (!arscp_target_exist())
@@ -481,7 +481,7 @@ arscp_replace(struct list *list)
 
 /* Rename the temporary archive to the target archive. */
 static void
-arscp_save()
+arscp_save(void)
 {
 	mode_t mask;
 
@@ -510,7 +510,7 @@ arscp_save()
  * invoking CREATE cmd on current archive.
  */
 static void
-arscp_clear()
+arscp_clear(void)
 {
 	char		*new_target;
 
@@ -546,7 +546,7 @@ arscp_end(int eval)
  * issued by user.
  */
 static int
-arscp_target_exist()
+arscp_target_exist(void)
 {
 
 	if (target)
@@ -621,7 +621,7 @@ arscp_mlist2argv(struct list *list)
 
 /* Free space allocated for argv array and its elements. */
 static void
-arscp_free_argv()
+arscp_free_argv(void)
 {
 	int i;
 
@@ -633,7 +633,7 @@ arscp_free_argv()
 
 /* Show a prompt if we are in interactive mode */
 static void
-arscp_prompt()
+arscp_prompt(void)
 {
 
 	if (interactive) {
